neqo-transport/src/recv_stream.rs

```diff
@@ -505,6 +505,8 @@ impl RecvStream {
                 let bytes_read = recv_buf.read(buf)?;
                 let fin_read = recv_buf.buffered() == 0;
                 if fin_read {
+                    self.conn_events
+                        .remove_stream_readable_events(self.stream_id);
```

Comment:
Let's put this call inside `set_state`? That's where we're already doing a similar thing.

Reply:
I was thinking about it but decided not to do it to avoid calling it too many times.
It is a better place, I will change it.